src/mapped-types.spec.ts

```diff
@@ -315,6 +315,8 @@ type RequiredOptionalProps = {
   // @dts-jest:pass:snap
   testType<DeepReadonly<NestedProps>['first']>();
   // @dts-jest:pass:snap
+  testType<DeepReadonly<DeepReadonly<NestedProps>['first']>>();
```

Comment:
Please don't add tests randomly into nicely ordered tests cases, add new ones at the end of the logical block

Reply:
got it, I put them in their own section, thanks.